libbeat/processors/ratelimit/docs/rate_limit.asciidoc

```diff
@@ -9,6 +9,9 @@ beta[]
 The `rate_limit` processor limits the throughput of events based on
 the specified configuration.
 
+In the current implementation, rate-limited events are dropped. Future implementations may allow rate-limited events to
+be handled differently.
+
```

Comment:
This change is unrelated to this PR but since it's minor I thought I would include it in here. Let me know if you'd prefer to have it in its own PR.